airflow/serialization/serialized_objects.py

```diff
@@ -319,6 +320,9 @@ def serialize_operator(cls, op: BaseOperator) -> dict:
         if op.operator_extra_links:
             serialize_op['_operator_extra_links'] = \
                 cls._serialize_operator_extra_links(op.operator_extra_links)
+        serialize_op['_templated_fields'] = {
+            field: serialize_template_field(getattr(op, field)) for field in op.template_fields
```

Comment:
Hmmmmm, not so sure about this -- specifically storing it under _templated_fileds, when this value here is different in content to the templated_fields on an non-serialized Operator.

Reply:
Removing Unrendered part from this PR. Will create a separate PR to add that